.github/workflows/respec.yaml

```diff
@@ -32,10 +34,10 @@ jobs:
     - name: Install dependencies
       run: npm ci
 
-    - uses: actions/checkout@v4 # checkout gh-pages branch
+    - uses: actions/checkout@v4
       with:
-        ref: gh-pages
-        path: deploy
+        ref: main
+        path: docs/deploy
```

Comment:
I think I kept this command only because it was creating the `deploy` folder for me. Of course, now that I am rethinking it doesn't make much sense. Fixed in https://github.com/OAI/OpenAPI-Specification/pull/3868/commits/e7a59992cfd4f34cf4d22eb322102622681bfb0a